python/lsst/ts/ATDome/mock_controller.py

```diff
@@ -108,19 +118,48 @@ class MockDomeController:
     * Encoder counts are not supported; reported values are bogus.
     * Door sequencing is not supported; doors move as if independent.
     """
-    def __init__(self, port, door_time=1, az_vel=6, home_az=10, home_az_overshoot=1, home_az_vel=1):
+
+    def __init__(
+        self,
+        port,
+        door_time=1,
+        az_vel=6,
+        home_az=10,
+        home_az_overshoot=1,
+        home_az_vel=1,
+    ):
         self.port = port
         self.door_time = door_time
         self.az_vel = Angle(az_vel, u.deg)
+        self.high_speed = Angle(5, u.deg)
+        self.coast = Angle(0.2, u.deg)
```

Comment:
currently set to 0.5

Reply:
OK. Note that the simulator is not meant to be realistic, just plausible. But I'll update it.